modules/britepoolIdSystem.md

```diff
@@ -0,0 +1,42 @@
+## BritePool User ID Submodule
+
+BritePool User ID Module. For assistance setting up your module please contact us at [prebid@britepool.com](prebid@britepool.com).
+
+### Prebid Params
+
+Individual params may be set for the BritePool User ID Submodule. At least one identifier must be set in the params.
+```
+pbjs.setConfig({
+    usersync: {
+        userIds: [{
+            name: ’britepoolId’,
+            storage: {
+                name: ‘britepoolid’,
+                type: ‘cookie’,
+                expires: 30
+            },
+            params: {
+                url: 'https://sandbox-api.britepool.com/v1/britepool/id', // optional
+                api_key: ’xxx’, // provided by britepool
+                hash: ’yyyy’, // example identifier
+                ssid: 'r894hvfnviurfincdejkencjcv' // example identifier
```

Comment:
Can you update the (majority) of the single quotes used around the values here?  They look like back-tics (`) rather than single quotes (')?

Reply:
Fixed.

Reply:
Also I've added a sandbox working api_key and identifiers in the britepoolIdSystem.md